### DISCUSSION TITLE
How to Setup

### DISCUSSION THREAD

**Discussion by SatisfactoryBug** [legendary-gl/legendary#237] [General]
I a noob and want to Try/use this without Epic Launcher as Epic crazy 800+ MB [Not Inc Games] 

1. I download legendary.exe
2. Then i put it into C:\Program Files (x86)\Legendary\
3.  I added firewall [If firewall needed ?] for legendary.exe
4. I Run legendary.exe the CMD did Pop-Up then Disappear
5. Any Help ?
Thank You

Oh, using on Windows 10 x64-Bit

**Comment by CommandMC** [comment]
Legendary is a command line program. It is not meant to be ran by the user double-clicking it.

You do not have to place the .exe file into any specific directory, it will work wherever. To use it, open up a CMD window in the directory it's in (or anywhere if you placed it into a directory in your `%PATH%`) and type in `legendary`

**Reply by SatisfactoryBug** [reply]
To CommandMC for about the CMD Thank You so any Help for next ?

so next i did [that i saw in Reddit so next i did:
1. legendary.exe auth
2. I got Website Pop-Up i copy / Paste with the "xxx":
{"redirectUrl":"https://epicgames.com/account/personal","sid":"xxx"}
3. I only copy the xxx 
4. I think it Error so any idea why??

C:\legendary>legendary.exe auth
[cli] INFO: Testing existing login data if present...
Please login via the epic web login!
If web page did not open automatically, please manually open the following URL: https://www.epicgames.com/id/login?redirectUrl=https://www.epicgames.com/id/api/redirect
Please enter the "sid" value from the JSON response: xxx
Traceback (most recent call last):
  File "urllib3\connection.py", line 169, in _new_conn
  File "urllib3\util\connection.py", line 96, in create_connection
  File "urllib3\util\connection.py", line 86, in create_connection
OSError: [WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "urllib3\connectionpool.py", line 699, in urlopen
  File "urllib3\connectionpool.py", line 382, in _make_request
  File "urllib3\connectionpool.py", line 1010, in _validate_conn
  File "urllib3\connection.py", line 353, in connect
  File "urllib3\connection.py", line 181, in _new_conn
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x000001A049D7CFA0>: Failed to establish a new connection: [WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "requests\adapters.py", line 439, in send
  File "urllib3\connectionpool.py", line 755, in urlopen
  File "urllib3\util\retry.py", line 573, in increment
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.epicgames.com', port=443): Max retries exceeded with url: /id/api/set-sid?sid=af21566623bd42f3a512bf812a4adab6 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001A049D7CFA0>: Failed to establish a new connection: [WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "legendary\cli.py", line 1375, in <module>
  File "legendary\cli.py", line 1335, in main
  File "legendary\cli.py", line 125, in auth
  File "legendary\core.py", line 109, in auth_sid
  File "requests\sessions.py", line 555, in get
  File "requests\sessions.py", line 542, in request
  File "requests\sessions.py", line 655, in send
  File "requests\adapters.py", line 516, in send
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.epicgames.com', port=443): Max retries exceeded with url: /id/api/set-sid?sid=af21566623bd42f3a512bf812a4adab6 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001A049D7CFA0>: Failed to establish a new connection: [WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions'))
[5656] Failed to execute script cli

C:\legendary>

**Reply by CommandMC** [reply]
Hm. Just by chance, you didn't follow any tutorials about Fortnite and using Easy Anticheat, did you? Just asking since we got a lot of people here because of that recently.

**Reply by SatisfactoryBug** [reply]
I hope i dont get banned for Link so this the link i saw at Reddit

https://www.reddit.com/r/EliteDangerous/comments/jz5otf/run_epic_games_version_of_ed_without_opening_the/

from Chermy_Mobix message

**Reply by CommandMC** [reply]
Hm, the instructions posted seem to make sense to me. Would you mind hopping over onto our Discord to get to the bottom of this? Github isn't really the best place for discussions IMO

**Reply by SatisfactoryBug** [reply]
That be fine

any link if allowed to add to discord ?

**Reply by CommandMC** [reply]
https://discord.gg/UJKBwPw

The link is also in the Readme here, right below the logo

**Reply by dodspringer** [reply]
> Legendary is a command line program. It is not meant to be ran by the user double-clicking it.
> 
> You do not have to place the .exe file into any specific directory, it will work wherever. To use it, open up a CMD window in the directory it's in (or anywhere if you placed it into a directory in your `%PATH%`) and type in `legendary`

assume I'm a normal person and I don't know what you mean by %PATH%.

Which Path? To what? Epic? The game I want to run? MS Paint?

**Comment by ghost** [comment]
I have the same problem.

It says the following:

requests.exceptions.SSLError: HTTPSConnectionPool(host='www.epicgames.com', port=xxx): Max retries exceeded with url: /id/api/set-sid?sid=xxx (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1108)')))
[12556] Failed to execute script cli

What should I do?

**Reply by derrod** [reply]
That sounds like you might have an anti-virus that does Man-in-the-middle attacks on your network traffic. Disable such features and throw it away.
